src/pages/index.js

```diff
@@ -2,56 +2,117 @@ import FormValidator from "../components/FormValidator.js";
 import Card from "../components/Card.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
-import { initialCards, config } from "../utils/constants.js";
+import { config } from "../utils/constants.js";
+import Api from "../components/Api.js";
 import "./index.css";
 
 //wrappers
 const profileEditModal = document.querySelector("#profile-edit-modal");
 const profileEditForm = profileEditModal.querySelector(".modal__form");
 const addCardModal = document.querySelector("#add-card-modal");
 const addCardForm = addCardModal.querySelector(".modal__form");
+const cardSelector = "#card-template";
```

Comment:
`const cardSelector = "#card-template";
`
From the 8th sprint all constants (`cardSelector ` and `settings`) should be in file `constants.js` in folder `utils` to clean up the main file.